src/app/components/Transactions/LogEvent.tsx

```diff
@@ -14,8 +14,15 @@ import { AccountLink } from '../Account/AccountLink'
 import { CopyToClipboard } from '../CopyToClipboard'
 import { TransactionLink } from './TransactionLink'
 import { SearchScope } from '../../../types/searchScope'
+import { AddressSwitchOption } from '../AddressSwitch'
+import { getOasisAddress } from '../../utils/helpers'
 
-const EvmEventParamData: FC<{ scope: SearchScope; param: EvmEventParam }> = ({ scope, param }) => {
+const EvmEventParamData: FC<{
+  scope: SearchScope
+  param: EvmEventParam
+  address?: string
+  addressSwitchOption: AddressSwitchOption
```

Comment:
Did you mean to use addressSwitchOption somewhere in here?

Reply:
Not, nice catch.